testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergRenameTable.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.iceberg;
+
+import io.trino.tempto.ProductTest;
+import io.trino.testng.services.Flaky;
+import org.testng.annotations.Test;
+
+import static io.trino.tempto.assertions.QueryAssert.Row.row;
+import static io.trino.tempto.assertions.QueryAssert.assertThat;
+import static io.trino.tests.product.TestGroups.ICEBERG;
+import static io.trino.tests.product.TestGroups.STORAGE_FORMATS;
+import static io.trino.tests.product.hive.util.TemporaryHiveTable.randomTableSuffix;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class TestIcebergRenameTable
+        extends ProductTest
+{
+    @Test(groups = {ICEBERG, STORAGE_FORMATS})
+    @Flaky(issue = "https://github.com/trinodb/trino/issues/4864", match = "Failed to read footer of file")
```

Comment:
did you see flakiness? I think we need to remove flaky tag for this issue at all places since it looks similar to issue fixed by #8504. I can take that in a separate PR, but we can remove it here if it's a copy paste.

Reply:
no, just copied from TestIcebergCreateTable. i only checked that https://github.com/trinodb/trino/issues/4864 is still open.

Reply:
removed